Sources/Jobs/JobsProvider.swift

```diff
@@ -4,17 +4,34 @@ import NIO
 
 /// A provider used to setup the `Jobs` package
 public struct JobsProvider: Provider {
-    
     /// See `Provider`.`register(_ services:)`
     public func register(_ s: inout Services) {
-        s.register(
-            QueueService.self
-        ) { container -> QueueService in
-            let persistenceLayer = try container.make(JobsPersistenceLayer.self)
-            
-            return QueueService(refreshInterval: self.refreshInterval,
-                                persistenceLayer: persistenceLayer,
-                                persistenceKey: self.persistenceKey)
+        s.register(QueueService.self) { container in
+            return try QueueService(
+                refreshInterval: self.refreshInterval,
+                persistenceLayer: container.make(JobsPersistenceLayer.self),
+                persistenceKey: self.persistenceKey
+            )
+        }
+
+        s.register(JobsConfiguration.self) { container in
+            return JobsConfiguration()
+        }
+
+        s.register(JobsCommand.self) { c in
+            return try .init(
+                queueService: c.make(),
+                jobContext: c.make(),
+                configuration: c.make()
+            )
+        }
+
+        s.register(JobContext.self) { c in
+            return .init(eventLoop: c.eventLoop)
+        }
+
+        s.extend(CommandConfiguration.self) { configuration, c in
+            try configuration.use(c.make(JobsCommand.self), as: "jobs")
```

Comment:
Let’s make this configurable as a parameter (the name of the command)

Reply:
This is related to #26, let's take it when we tackle that.